commands/src/main/java/org/wildfly/extras/creaper/commands/orb/Attribute.java

```diff
@@ -0,0 +1,82 @@
+package org.wildfly.extras.creaper.commands.orb;
+
+
+/**
+ * Definition of attribute transition state. As builder defines
+ * it could be
+ * <ul>
+ *   <li>having value - current value will be redefined</li>
+ *   <li>having no value (being null) - leaving current value where it is</li>
+ *   <li>undefined value - current value will be undefined</li>
+ * </ul>
+ */
+final class Attribute<T> {
+    private final T value;
+    private final boolean isToUndefine;
+
+    private Attribute(T value) {
+        this.value = value;
+        this.isToUndefine = false;
+    }
+
+    private Attribute(T value, boolean isToUndefine) {
+        this.value = value;
+        this.isToUndefine = isToUndefine;
+    }
+
+    public static <T> Attribute<T> noValue() {
+        return new Attribute<T>(null);
+    }
+
+    public static <T> Attribute<T> undefine() {
+        return new Attribute<T>(null, true);
+    }
+
+    public static <O> Attribute<O> undefine(O object) {
```

Comment:
Hmm, I wonder what is this good for? The `object` is never used in this method. And the method seems to be unused as well.

If you wanted a type-safe method and the `<T> Attribute<T> undefine()` variant isn't enough (which might happen in Java pre 8), idiomatic solution would be `<T> Attribute<T> undefine(Class<? extends T> clazz)`.